packages/cpf-generator/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "cpf-generator",
```

Comment:
What do you think about replacing "cpf-generator" to "generate-cpf"?

Reply:
I think the package should follow the same name conventions we use in our other packages.
`@brazilian-utils/cpf-generator`

Reply:
Maybe @brazilian-utils/generate-cpf @andreoav ?

Reply:
for me the package name can be this too. maybe will fits better with the other packages name